images/linux/scripts/installers/nodejs.sh

```diff
@@ -4,14 +4,27 @@
 ##  Desc:  Installs Node.js LTS and related tooling (Gulp, Grunt)
 ################################################################################
 
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
 # Install LTS Node.js and related build tools
 curl -sL https://raw.githubusercontent.com/mklement0/n-install/stable/bin/n-install | bash -s -- -ny -
 ~/n/bin/n lts
-npm install -g grunt gulp n parcel-bundler typescript newman vercel
-npm install -g --save-dev webpack webpack-cli
+# Install node modules
+node_modules=$(get_toolset_value '.node_modules[]')
+
+# remove commenting out of this line as nmp migrated to v7
+# npm install -g $node_modules
 
-# Install the Netlify CLI using --unsafe-perm=true options to avoid permission issues
-npm install -g --unsafe-perm=true netlify-cli
+# TODO: workaround for Netlify CLI with npm6. Remove 19-28 and uncomment 17 if migration to npm7 compelted
+for module in $node_modules; do
+    echo "Installing node module $module"
+    if [ $module = "netlify-cli" ];then
+      npm install -g --unsafe-perm=true $module
```

Comment:
please, restore comment for netlify installation flag `# Install the Netlify CLI using --unsafe-perm=true options to avoid permission issues`